src/sys/mod.rs

```diff
@@ -61,13 +62,17 @@ cfg_os_poll! {
     }
 
     cfg_udp! {
-        pub(crate) use self::unix::UdpSocket;
+        pub(crate) use self::unix::udp;
     }
 
     cfg_uds! {
         pub(crate) use self::unix::{UnixDatagram, UnixListener, UnixStream};
         pub use self::unix::SocketAddr;
     }
+
+    cfg_any_os_util! {
```

Comment:
`IoSourceState` is also only used by the UDP features in this change, so this should be changed to `cfg_udp!`

Reply:
I've changed this to `cfg_net!`.